notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>Do &gt; Due</p><p>upto &gt; up to</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1570/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>